### DISCUSSION TITLE
In Hybrid Search, the file summary query does not work

### DISCUSSION THREAD

**Discussion by tjdalsry** [open-webui/open-webui#4330] [General]
# Bug Report

## Description

**Bug Summary:**
**In Hybrid Search, the file summary query does not work**

When Hybrid Serach is turned off, the original function works. However, when turned on, it works.

When you check the debugging, it embeds well.

**Steps to Reproduce:**

1. Attach the document.

2. chat to "summarize this file"

3. "No files provided." are answered.

**Expected Behavior:**

Detailed Summary of Files

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Open WebUI Version: 0.37
- **Ollama (if applicable): 0.1.47

- **Operating System: Ubuntu 22.04

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**

<img width="772" alt="화면 캡처 2024-08-03 220427" src="https://github.com/user-attachments/assets/e2c77776-fa29-4615-a44d-b734edbc73f9">

**Comment by silentoplayz** [comment]
This oddly appears to be an issue with the model being used (Qwen 2). The model can't tell me anything about documents either, but other models can fine. Even CodeQwen works well with RAG. It can only tell me about the RAG template esentially; the 7b and 14b models of Qwen 2. The 0.5b model only worked sorta once with RAG.

**Reply by tjdalsry** [reply]
It's not a problem with the model, it seems like there's a problem with the logic.
Because of the hybrid search, it is designed to bring a document similar to the query, so I don't think I can bring the contents.

**Reply by oatmealm** [reply]
For me only the first document I load is visible to the rag. Everything else sometimes gets embedded, based on the logs, but the model always returns no file available. I've tried it with different models, including llama 3.1 and different embedding, including ollama/openai etc. doesn't work.

**Comment by Shadowfita** [comment]
I am having the same issue when hybrid search is enabled. No context from RAG is provided to the LLM.
